timelock-server/src/integTest/java/com/palantir/atlasdb/timelock/TestableTimelockCluster.java

```diff
@@ -74,7 +74,10 @@ public TestableTimelockCluster(String baseUri, String defaultClient, String... c
     }
 
     public void waitUntilLeaderIsElected() {
```

Comment:
I think we should get rid of this, and force everyone to specify an agent.

Reply:
Went the other way so you cannot specify the default agent, which is a random uuid